core/templates/dev/head/library/library.html

```diff
@@ -56,16 +56,16 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
                translate="I18N_LIBRARY_NO_EXPLORATION_GROUPS">
           </div>
 
-          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
+          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index" ng-init="groupIndex = $index">
```

Comment:
@seanlip  The issue with $index is, It takes the index of the recent ng-repeat. <span ng-if="currentlyHighlightedGroupIndex === $index"> will give me the index of the tile and not the group. That is the reason why I used the ng-init. I will check if there is an alternative.